pkcs5/src/pbes2/encryption.rs

```diff
@@ -48,6 +49,71 @@ fn cbc_decrypt<'a, C: BlockCipherDecrypt + BlockCipher + KeyInit>(
         .map_err(|_| Error::EncryptFailed)
 }
 
+fn gcm_encrypt<'a, C>(
+    es: EncryptionScheme,
+    key: EncryptionKey,
+    iv: &[u8],
```

Comment:
```suggestion
    iv: Nonce<C>,
```

from: https://docs.rs/aead/0.5.2/aead/type.Nonce.html